pkg/cmd/operator/analyze.go

```diff
@@ -77,7 +83,7 @@ func (o *AnalyzeOptions) Validate() error {
 
 	errs = append(errs, o.ClientConfig.Validate())
 
-	if len(o.ArchivePath) > 0 {
+	if o.IsArchivePathSet() {
```

Comment:
DRY. reverted

Reply:
It's good to unify how you check for non-emptiness, but I'd say this is a bit of a misunderstanding of the DRY principle, since the comparison is idiomatic, and the method is syntactic sugar at best.
It would be good to abstract this if checking for the input "mode" required some more logic, but in that case you would probably want to set an additional attribute in `Complete()` either way, instead of invoking the function every time.